lib/TPP/Dialect/Tpp/TppUtils.cpp

```diff
@@ -308,15 +308,49 @@ bool canMapToTppAdd(linalg::GenericOp linalgOp) {
   return hasOnlyScalarElementwiseOp<arith::AddFOp>(linalgOp.getRegion());
 }
 
-// TODO: check access pattern using affine map.
+// Return true if the linalgOp has only identity maps and its operands are
+// shapedType with rank <= 2.
+static bool checkOperandsAndMaps(linalg::GenericOp linalgOp) {
```

Comment:
Bad name for this function, doesn't reflect what it does. Comment is not visible at the call site.